pages/common/mogrify.md

```diff
@@ -0,0 +1,23 @@
+# Mogrify
+
+> Tool that allows making operations on multiple images ie. resizing, cropping, flipping, adding effects. Replaces the original file.
```

Comment:
OK, I'm not english native, if you think it fits better, I believe you, I'll also learn something ;)